build.gradle.kts

```diff
@@ -9,6 +14,75 @@ allprojects {
     group = findProperty("GROUP").toString()
 }
 
+subprojects {
+    tasks.withType<JavaCompile>().configureEach {
```

Comment:
Whenever a task which has the type of `JavaCompile` is used, these properties are configured upon it (namely, we set the version of Java to use).